tests/unit/compiler/venom/test_memmerging.py

```diff
@@ -483,30 +562,10 @@ def test_memmerging_allowed_overlapping():
     _check_pre_post(pre, post)
 
 
-def test_memmerging_allowed_overlapping2():
```

Comment:
@HodanPlodky can you give the reasoning for removing this test

Reply:
note: re-added here https://github.com/vyperlang/vyper/pull/4422/commits/00eea3c9feb36c379d37cfd5f7937f6fceb23fbd